themes/stanford_basic/src/scss/components/hero/_hero.scss

```diff
@@ -0,0 +1,9 @@
+@charset "UTF-8";
+
+// Banner aspect ratio
+
+.su-hero__media {
+  @include grid-media-max('sm') {
+    aspect-ratio: 30/11;
```

Comment:
Interesting ratio. 3/1 would've been just about the same. but it works.

Reply:
Yeah that's what our [userguide](https://sitesuserguide.stanford.edu/build-and-design/media-library/images) has for the banner aspect ratio, so went with that. But agreed, about the same.